src/index.html

```diff
@@ -64,36 +64,132 @@ <h2 class="community-title">Join the Community</h2>
     <section class="gallery">
       <h2 class="gallery-title">Our Meetups Gallery</h2>
 
+      <!-- ----- Gallery Modal ----- -->
+      <div class="gallery-modal">
+        <!-- Meetup August -->
+        <div class="card-modal" id="meetup-august">
+          <div class="modal-container">
+            <img
+              class="modal-photo"
+              src="images/meetup-august.jpg"
+              alt="meetup-august"
+            />
+            <div class="modal-content">
+              <div>
+                <p class="modal-title">August RoundTable Meetup</p>
+                <p class="modal-text">
+                  <img class="map-icon" src="images/map-pin.svg" /> Bo's Coffee,
+                  Ayala
+                </p>
+              </div>
+              <a href="#modal-close">Close</a>
+            </div>
+          </div>
+        </div>
+
+        <!-- Meetup July -->
+        <div class="card-modal" id="meetup-july">
+          <div class="modal-container">
+            <img
+              class="modal-photo"
+              src="images/meetup-july.jpg"
+              alt="meetup-july"
+            />
+            <div class="modal-content">
+              <div>
+                <p class="modal-title">July RoundTable Meetup</p>
+                <p class="modal-text">
+                  <img class="map-icon" src="images/map-pin.svg" /> Coffee
+                  Project, Mandaluyong
+                </p>
+              </div>
+              <a href="#modal-close">Close</a>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
I think this is an extra closing `</div>`

Reply:
owh!! my bad. It's fixed now. Thanks!